api/src/main/java/org/openmrs/module/fhir2/api/translators/impl/ObservationTranslatorImpl.java

```diff
@@ -80,6 +85,8 @@ public Observation toFhirResource(Obs observation) {
 		
 		obs.setValue(observationValueTranslator.toFhirResource(observation));
 		
+		obs.setInterpretation(Collections.singletonList(interpretationTranslator.toFhirResource(observation)));
```

Comment:
`addInterpretation()` rather than `setInterpretation()` here.

Reply:
OpenMrs only allows adding a single interpretation to an observation, **addInterpretation()** is not supported

Reply:
So, I'm not suggesting we support multiple interpretations, just that `obs.addInterpretation(interpretationTranslator.toFhirResource(observation))` does the same thing and is a bit shorter and clearer.